<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Guide</class>
 <widget class="QWidget" name="Guide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>133</width>
    <height>178</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="tabLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="7" column="0">
        <widget class="QCheckBox" name="darkFrameCheck">
         <property name="toolTip">
          <string>Subtract dark frame. If no dark frame is available, a new dark frame shall be captured and saved for future use.</string>
         </property>
         <property name="text">
          <string>Dark</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="guiderCombo"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="binningCombo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="toolTip">
          <string>Select which device receives the guiding correction commands.</string>
         </property>
         <property name="text">
          <string>Via:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Guider:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="toolTip">
          <string>Guide camera binning. It is recommended to set binning to 2x2 or higher.</string>
         </property>
         <property name="text">
          <string>Bin:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>Exposure time in seconds</string>
         </property>
         <property name="text">
          <string>Exp:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="boxSizeCombo">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="toolTip">
          <string>Apply filter to image after capture to enhance it</string>
         </property>
         <property name="text">
          <string>Filters:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="toolTip">
          <string>Guide star tracking box size. Box size must be set in accordance to the selected star size.</string>
         </property>
         <property name="text">
          <string>Box:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="filterCombo">
         <item>
          <property name="text">
           <string>--</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="exposureIN">
         <property name="maximum">
          <double>60.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="ST4Combo"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
